apiserver/facades/agent/lifeflag/facade.go

```diff
@@ -30,7 +62,7 @@ func NewFacade(backend Backend, watcherRegistry facade.WatcherRegistry, authoriz
 			return authorizer.AuthOwner(tag)
 		}, nil
 	}
-	life := common.NewLifeGetter(backend, getCanAccess)
+	life := common.NewLifeGetter(applicationService, machineService, backend, getCanAccess, logger)
```

Comment:
We might want to consider removing the life getter at some point. Many facade's life getters only get the life gor particular types of entities, yet they need services for many.

Not something we need to rush though